src/pages/user/UserOrcid.tsx

```diff
@@ -64,6 +66,13 @@ const UserOrcid: FC = () => {
     setOpenConfirmDialog(!openConfirmDialog);
   };
 
+  useEffect(() => {
+    const orcidError = new URLSearchParams(location.hash.replace('#', '?')).get('error');
```

Comment:
usikker på om jeg skjønner dette... men hvis det går skeis redirecter ORCID til oss med `#error=access_denied` i URL, så må vi erstatte `#` med `?` får å kunne bruke `get()` på et gyldig `URLSearchParams`-objekt?

Reply:
Ja, det stemmer. Har lett etter en annen måte å gjøre det på, men finner ikke noe enklere enn å bruke URLSearchParams 🤔 har du peiling på noe annet som kan gjøres?

Reply:
`location.hash.split("#").pop()` vil sikkert gjøre det samme, men vet ikke om det egentlig er så mye bedre heller

Reply:
jo, synes det er litt bedre - går for det jeg 👍

Reply:
nei, den vil jo selvsagt få med "error"... `location.hash.split("#error=").pop()` da :D (huff)

Reply:
får med dette fra orcid: 
`http://localhost:3000/user#error=access_denied&error_description=User%20denied%20access`

så tror det er best å kjøre en kombo:
`const orcidError = new URLSearchParams(location.hash.split('#').pop()).get('error');`

hva tror du om det?

Reply:
men blir det egentlig bedre? 😛

Reply:
haha, fysj. kan bare la det stå som det var